openpype/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,225 @@
+        label="Settings Defined Arguments",
+        disabled=False
+    ))
+
+    # Only display Boolan flags that the Admin defined as overrideable
+    abc_boolean_overrides = [
+        arg
+        for arg in abc_boolean_args
+        if arg in abc_args_overrides
+    ]
+
+    abc_defs.append(EnumDef(
+        "abcExportBooleanArguments",
+        abc_boolean_overrides if abc_boolean_overrides else [""],
+        multiselection=True,
+        label="Arguments Overrides",
+        disabled=True if not abc_boolean_overrides else False
+    ))
+
+    abc_defs.append(
+        TextDef(
+            "attr",
+            label="Custom Attributes",
+            default=getattr(cls, "attr", None),
+            placeholder="attr1; attr2; ...",
+            disabled=True if "attr" not in abc_args_overrides else False,
+        )
+    )
+
+    abc_defs.append(
+        TextDef(
+            "attrPrefix",
+            label="Custom Attributes Prefix",
+            default=getattr(cls, "attrPrefix", None),
+            placeholder="prefix1; prefix2; ...",
+            disabled=True if "attrPrefix" not in abc_args_overrides else False,
+        )
+    )
+
+    abc_defs.append(
+        EnumDef(
+            "dataFormat",
+            label="Data Format",
+            default=getattr(cls, "dataFormat", None),
+            items=["ogawa", "HDF"],
+            disabled=True if "dataFormat" not in abc_args_overrides else False,
+        )
+    )
+
+    abc_defs.append(
+        NumberDef(
+            "preRollStartFrame",
+            label="Start frame for preroll",
+            default=getattr(cls, "preRollStartFrame", None),
+            tooltip=(
+                "The frame to start scene evaluation at. This is used to set"
+                " the starting frame for time dependent translations and can"
+                " be used to evaluate run-up that isn't actually translated."
+            ),
+            disabled=True
+            if "preRollStartFrame" not in abc_args_overrides
+            else False,
+        )
+    )
+
+    return abc_defs
+
+
+class CreateAnimation(plugin.MayaHiddenCreator):
+    """Animation output for character rigs
+
+    We hide the animation creator from the UI since the creation of it is
+    automated upon loading a rig. There's an inventory action to recreate it
+    for loaded rigs if by chance someone deleted the animation instance.
+    """
+
+    identifier = "io.openpype.creators.maya.animation"
+    name = "animationDefault"
+    label = "Animation"
+    family = "animation"
+    icon = "male"
+
+    write_color_sets = False
+    write_face_sets = False
+    include_parent_hierarchy = False
+    include_user_defined_attributes = False
+
+    def collect_instances(self):
+        pass
+
+    def get_instance_attr_defs(self):
+        super(CreateAnimation, self).get_instance_attr_defs()
+        # adding project settings, since MayaHiddenCreator does not
+        # handle this for us (yet?)
```

Comment:
Can you double check whether this is actually resolved since this PR got merged: https://github.com/ynput/OpenPype/pull/5908

I suspect that would've fixed it.

Reply:
YEah, this is resolved, forgot to remove the comment, but last develop merge fixes it